<?xml version="1.0"?>
<!-- EXPERIMENTAL - brand new driver -->
<launch>
    <!-- robot_description and robot_state_publisher will be published here -->
  <node name="sick_mrs_6xxx" pkg="sick_scan" type="sick_generic_caller" respawn="false" output="screen">
    <!-- default values: -->
    <!--
      <param name="scanner_type" type="string" value="sick_mrs_6xxx"
      <param name="min_ang" type="double" value="-2.35619449019" />
      <param name="max_ang" type="double" value="2.35619449019" />
      <param name="intensity" type="bool" value="True" />
      <param name="skip" type="int" value="0" />
      <param name="frame_id" type="str" value="laser" />
      <param name="time_offset" type="double" value="-0.001" />
      <param name="publish_datagram" type="bool" value="False" />
      <param name="subscribe_datagram" type="bool" value="false" />
      <param name="device_number" type="int" value="0" />
      <param name="range_min" type="double" value="0.05" />
    -->
    <param name="scanner_type" type="string" value="sick_mrs_6xxx"/>
    <param name="range_max" type="double" value="25.0" />
    <param name="hostname" type="string" value="192.168.0.24" />
    <param name="port" type="string" value="2112" />
    <param name="timelimit" type="int" value="5" />
    <param name="min_ang" type="double" value="-1.047197333" />
    <param name="max_ang" type="double" value="+1.047197333" />


  </node>
</launch>
